src/Raven.Server/Documents/Handlers/Admin/AdminIndexHandler.cs

```diff
@@ -144,8 +141,14 @@ await using (var writer = new AsyncBlittableJsonTextWriter(context, ResponseBody
             }
         }
 
-        private async Task HandleLegacyIndexesAsync()
+        private async Task HandleIndexesFromLegacyReplicationAsync()
```

Comment:
Better yet, is there _any_ reason not to abort this feature entirely?
That is a legacy feature from pre 4.0 days, no?

Reply:
This is still being used.

Reply:
Who is using that?